src/app/landing/mentors/mentors.component.ts

```diff
@@ -1,14 +1,22 @@
-import { Component } from '@angular/core';
-import { dummyMentors } from './dummyMentors';
+import { Component, OnInit } from '@angular/core';
+import { MentorsService } from './mentors.service';
+import { Observable } from 'rxjs';
+import { Mentor } from '../shared/interfaces/mentor';
 
 @Component({
   selector: 'app-mentors',
   templateUrl: './mentors.component.html',
-  styleUrls: [
-    '../common.css',
-    './mentors.component.css'
-  ]
+  styleUrls: ['./mentors.component.css']
 })
-export class MentorsComponent {
-  mentors = dummyMentors
+export class MentorsComponent implements OnInit{
+
+  mentors$: Observable<Mentor[]> | undefined = undefined
+
+  constructor(private mentorsService: MentorsService){
+  }
+
+  ngOnInit(): void {
+    this.mentors$ = this.mentorsService.getMentors()
```

Comment:
you can define it in `line 13`

Reply:
ok, changed it in the same commit with **stutents-feedback.component.ts**